### DISCUSSION TITLE
Integration with Google Calendar

### DISCUSSION THREAD

**Discussion by aaqsr** [AppFlowy-IO/AppFlowy#329] [Ideas]
Hey you all, I just discovered this project and really love what you guys are doing here. And I wanted to request a feature if you don't mind.

I would love to see an integration with google calendar, where if I have reminders in the app, they show up on my calendar. This is the one main gripe I have with notion: it doesn't sync to a google calendar, so you can't use notion note reminders to schedule anything. Being able to import a calendar that syncs with my google calendar, or being able to embed the calendar into pages would also be amazing.

**Comment by annieappflowy** [comment]
Thanks for proposing this feature. I share the same use case with you. I've converted this into an issue that can be claimed by developers.

**Reply by wwwdyne** [reply]
@annieappflowy 

Google Calendar has recurring tasks support (out-of-the-box). So before implementing this task (Integration with Google Calendar), both these tasks should be implemented:

https://github.com/AppFlowy-IO/AppFlowy/issues/3437
https://github.com/AppFlowy-IO/AppFlowy/issues/2380

Could you please assign a qualified software engineer for it?

**Comment by Smagar0** [comment]
This is the most important missing feature Notion never added to their app

**Comment by BoBoat1** [comment]
I agree

**Comment by 7homR** [comment]
Just wondering if this is being actively pursued? I'm looking to switch to Appflowy from Notion, where I've already gone through the hassle of Gcal syncing through Zapier -- not all that keen on doing it again.

**Comment by Artemis1-0** [comment]
This is a great feature idea that would integrate perfectly with my current workflow. Maybe microsoft integration as well (I hope)

**Comment by EmilEmilchen** [comment]
I'd love this as well!

**Comment by AryanAtif** [comment]
We need this so much

**Reply by Artemis1-0** [reply]
Indeed we do.

**Comment by saxcorp** [comment]
We need this so much

**Comment by imhasnainizhar** [comment]
Now we need to start work on it

**Reply by imhasnainizhar** [reply]
Can anybody tell me how much is already done
